config/crd/bases/oadp.openshift.io_nonadminbackups.yaml

```diff
@@ -619,6 +619,17 @@ spec:
                 - Created
                 - Deleting
                 type: string
+              queueInfo:
+                description: |-
```

Comment:
Please also create the corresponding OADP Operator PR so that we merged the CRD changes first and CI passess here.

Reply:
https://github.com/openshift/oadp-operator/pull/1598